config/manager/manager.yaml

```diff
@@ -20,6 +20,7 @@ spec:
     controller-tools.k8s.io: "1.0"
   ports:
   - port: 443
```

Comment:
Is there a reason to keep port 443 around?

Reply:
Nope I don't believe so, just fixed that now. :)

Reply:
My mistake, it is required. If you wanted it to be 8443 here then the provisioning of the `ValidatingWebhookConfiguration` would need changing to provide a custom server port (defaults to 443 when not specified).